tests/models/decoder_only/language/test_mistral.py

```diff
@@ -58,17 +61,62 @@
             },
             "required": ["city", "state", "unit"]
         }
+    },
```

Comment:
Make test much more difficult, complex to show the community to what extent function calling can be used with Mistral models